packages/cli/src/commands/setup/auth/templates/dbAuth.function.ts.template

```diff
@@ -109,6 +109,8 @@ export const handler = async (
     //
     // If this returns anything else, it will be returned by the
     // `signUp()` function in the form of: `{ message: 'String here' }`.
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
If we do decide to just keep the comment, it should probably be changed to this, right?

```suggestion
    // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
```

Reply:
Really good catch, i actually briefly read through 3fe7dca57df8eaadda4185bfe6170b6733aebddc, but then forgot about it.

Actually, there's numerous places in the code wouldn't work for JS.

Will fix that also in a separate commit!

...actually, we should have a test for that, otherwise we can place bets on how long it takes until this regresses again (@jtoar ?) ;)

Reply:
→ 84f208917

Reply:
Yikes, found two more. Amended the commit now: 6cf723006